docker-compose-for-tests.yml

```diff
@@ -3,11 +3,10 @@ services:
   silo:
     image: ${SILO_IMAGE}
     volumes:
-      - ./testBaseData:/data
+      - ./testBaseData/exampleDataset:/data
     ports:
       - "8080:8081"
     command:
       - "--api"
-      - "--preprocessingConfig=./preprocessing_config_for_ci.yaml"
+      - "--preprocessingConfig=./preprocessing_config.yaml"
       - "--databaseConfig=./test_database_config.yaml"
```

Comment:
I just wanted to mention that we now have defaults for the config paths:
./preprocessing_config.yaml
and ./database_config.yaml
Which we might want to use here, but we can also showcase the configurability

Reply:
I'd showcase the configurability here, that's why I wanted to make it explicit. We might redesign the Docker images a little where we make use of the defaults :+1: